src/projectservice/index.js

```diff
@@ -70,9 +72,63 @@ app.post('/createProject', async (req, res) => {
   })
 })
 
+// Endpoint for deleting a project
+app.delete('/deleteProject/:projectID', async (req, res) => {
+  let token = req.body.token
+  if (!token) {
+    const { username, password } = req.body
+    token = await getToken(username, password)
+  }
+  const projectID = req.params.projectID
+  try {
+    const status = await deleteProject(projectID, token)
+    const data = {
+      // eslint-disable-next-line quote-props
+      'status': status.message,
+      // eslint-disable-next-line quote-props
+      'TaigaAPIResponseStatus': status.success
+    }
+    if (status.success) {
+      res.status(201).send(data)
```

Comment:
HTTP response code for delete verb should not be 201, it should be either 200 OK or 204 NO CONTENT.